centaur/src/main/resources/standardTestCases/sub_workflow_var_refs/sub_workflow_var_refs.wdl

```diff
@@ -7,13 +7,13 @@ workflow top_level_workflow {
   Array[String] pieces = ["hello", "lowly", "subject!"]
 
   scatter (i in is) {
-    call subworkflow.subhello as subhello {
+    call subworkflow.subhello as subhello_call_alias {
```

Comment:
Was this necessary?

It's possible that this was sneakily side-testing that an alias that matches the task name doesn't cause any problems?

Reply:
It was to make the contents of memory more obvious while debugging... no reason to keep, so reverting